.github/workflows/ci-tests.yml

```diff
@@ -91,13 +110,20 @@ jobs:
         tox -e ${{ env.TOXENV }}
   examples:
     runs-on: ubuntu-latest
+    env:
+      PYTHON_VERSION: '3.10'
     name: example tests
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
-        python-version: '3.10'
+        python-version: ${{ env.PYTHON_VERSION }}
+    - name: Cache dependencies
+      uses: actions/cache@v2
+      with:
+        path: ${{ env.pythonLocation }}
+        key: ${{ env.PYTHON_VERSION }}-${{ runner.os }}-build-${{ hashFiles('setup.py') }}
```

Comment:
This line is fine, but in line 130 remove `pip install tqdm` as it's installed as part of the package anyway.